docs/documentation/stories/universal-rendering.md

```diff
@@ -6,7 +6,7 @@ The Angular CLI supports generation of a Universal build for your application. T
 
 ## Example CLI Integration:
 
-[Angular Universal-Starter](https://github.com/angular/universal-starter/tree/master/cli) - Clone the universal-starter, and check out the `/cli` folder for a working example.
+[Angular Universal-Starter](https://github.com/angular/universal-starter) - Clone the universal-starter, and check out the `/cli` folder for a working example.
```

Comment:
The universal-starter is no longer split between `cli` and `webpack`. You can just say "Clone the universal starter for a working example."

Reply:
Good catch. Fixed.